src/screens/gm/objectCreationView.cpp

```diff
@@ -142,7 +142,10 @@ bool GuiObjectCreationView::onMouseDown(sf::Vector2f position)
 void GuiObjectCreationView::setCreateScript(string script)
 {
     create_script = script;
-    enterCreateMode();
+    gameGlobalInfo->on_gm_click = [this] (sf::Vector2f position)
```

Comment:
I see a potential problem here:
1) open gm screen
2) click create button
3) close gm screen
4) open new gm screen
5) click on screen, the on_gm_click references the old deleted gm screen in the lambda